src/main/java/uk/gov/pay/directdebit/common/exception/BadRequestException.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.pay.directdebit.common.exception;
+
+public class BadRequestException extends ApiException {
+
+    public BadRequestException(String message) {
+        super(400, message);
```

Comment:
Same here I would expect a BadRequestExceptionMapper knowing that when it happens it returns a 400